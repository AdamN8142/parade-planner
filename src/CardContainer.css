.card-container {
  display: grid;
  grid-template-columns: repeat(auto-fit, minmax(300px,1fr));
}

.card-photo {
  /* display: flex; */
  flex-wrap: flex;
  justify-content: space-between;
  height: 250px;
  width: 300px;
  margin-left: 10px;
  margin-right: 10px;
  opacity: .75;
}

.card-photo:hover {
  opacity: 1;
}

h3 {
  font-size: 16px;
  text-align: center;
  margin-bottom: 20px
}