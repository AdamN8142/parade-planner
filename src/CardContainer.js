import React, { Component } from 'react';
import Card from './Card'



class CardContainer extends Component {
  constructor(){
      super();
  }

    render(){
      return(
          <div>
                WHAT ARE YOU DOING  
          </div>
      )
  }     
}



export default CardContainer;