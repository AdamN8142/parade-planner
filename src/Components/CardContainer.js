import React, { Component } from 'react';
import Card from './Card'

class CardContainer extends Component {
  constructor(){
      super();
  }

    render() {
      return(
          <div>
              <Card />
          </div>
      )
  }     
}



export default CardContainer;