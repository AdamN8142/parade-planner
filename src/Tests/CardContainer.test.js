import React from 'react';
import CardContainer from '../Components/CardContainer';
import { shallow } from 'enzyme';

describe('App', () => {

  let wrapper;

  beforeEach(() => {
    wrapper = shallow(
      <CardContainer />
    )
  });
	
  it('should match the snapshot with all data passed in', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('should have a proper default state', () => {
    expect(wrapper.state()).toEqual({ id: -1, isExpanded: false });
  });

  // testing toggleCard 
  it('should change state', () => {
    
  });

  // testing displayCards
  it('should have a proper default state', () => {

  });
});