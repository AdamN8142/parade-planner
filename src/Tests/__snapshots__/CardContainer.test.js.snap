// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should match the snapshot with all data passed in 1`] = `
<div>
  <section
    className="card-container"
  >
    <article
      className="container"
      name={1}
    >
      <div
        className="individual-card"
        name={1}
      >
        <img
          className="card-photo"
          name={1}
          onClick={[Function]}
          src="https://assets.simpleviewcms.com/simpleview/image/fetch/c_limit,q_75,w_1200/https://assets.simpleviewinc.com/simpleview/image/upload/crm/jefferson/37872354-littlerascals2-jpg0_cc32794e-5056-a36a-08469bbb1ddece75.jpg"
        />
        <h3
          className="card-title"
          name={1}
        >
           
          Krewe of Little Rascals
           
        </h3>
      </div>
    </article>
    <article
      className="container"
      name={2}
    >
      <div
        className="individual-card"
        name={2}
      >
        <img
          className="card-photo"
          name={2}
          onClick={[Function]}
          src="https://www.hlimg.com/images/events/738X538/25mardi6502_1546063931e.jpg"
        />
        <h3
          className="card-title"
          name={2}
        >
           
          Krewe of Excalibur
           
        </h3>
      </div>
    </article>
    <article
      className="container"
      name={3}
    >
      <div
        className="individual-card"
        name={3}
      >
        <img
          className="card-photo"
          name={3}
          onClick={[Function]}
          src="https://d23xispzx43ico.cloudfront.net/wp-content/uploads/2015/02/0212-babylon-knights.jpg"
        />
        <h3
          className="card-title"
          name={3}
        >
           
          Krewe of Babylon
           
        </h3>
      </div>
    </article>
  </section>
</div>
`;
